src/history/api.py

```diff
@@ -3,16 +3,17 @@
 from datetime import datetime
 from pathlib import Path
 from typing import Union
+import re
 
 ROUNDS = 4
 INF = 2147483647
 NA = -1
 BASE_DIR = Path(Path(__file__).resolve()).parent
 
-
+YEAR_REGEX = re.compile("[0-9]{8}")
 def _clean_year(year: Union[str, int]) -> str:
     """
-    Clean the string to YYYY.
+    Comments can either be "YYYY[ /-]YYYY" or "YYYY".
```

Comment:
I think that saying that "The string" can either be... or "The year string" makes more sense than "Comments" here